src/useActiveOrg.tsx

```diff
@@ -1,36 +0,0 @@
-import { useContext } from "react"
-import { AuthContext } from "./AuthContext"
-
-const ORG_SELECTION_LOCAL_STORAGE_KEY = "__last_selected_org"
-
-export function useActiveOrg() {
```

Comment:
So unfortunately, we can't remove this without it being a breaking change (we also can't name the new hook useActiveOrg). This should also be marked as deprecated and we can come up with another name for the new hook. 

There might be a middleground where we make it so useActiveOrg works with existing and new integrations, something like:

- If someone specified an activeOrgFn, useActiveOrg should behave like this (with a warning)
- If someone didn't specify an activeOrgFn, we have to worry about if they were using the default function which was backed by localStorage. If we re-use the same localstorage location (and store the ID instead of what I suggested in my other comment), it would work the same.
- Otherwise, new integrations would use the set function which the other hook would work with.

We can chat more about the tradeoffs of that vs just deprecating this offline?

Reply:
Let's chat more about this offline :)